src/View.ts

```diff
@@ -646,11 +646,13 @@ export class ViewWrapper extends EventHandler {
       });
 
       const groupsArray = Array.from(groups);
-      const categoryOrder = getPlugin('chooserConfig', 'chooser_header_order');
+
+      const orderDescs = listPlugins('chooserConfig').map((desc) => desc.order);
```

Comment:
Currently there is no default order. But I think I'll order the groups alphabetically, if no order is given (as it would be the same behavior as suggested in Caleydo/targid_boehringer#449)